docs/platform/howto/set-authentication-policies.md

```diff
@@ -16,29 +13,55 @@ address and password. For an added layer of security, you can enforce
 two-factor authentication (2FA) for password logins for all users in
 your organization.
 
-When 2FA is required, users won't be able to
-access any resources in your organization until they set up 2FA.
+When 2FA is required, users can't access any resources in your organization until they
+set up 2FA.
 
 :::note
-Authentication tokens are not affected and continue to work when you make 2FA required.
-However, when users [enable 2FA](/docs/platform/howto/user-2fa) their existing authentication tokens are revoked.
+Personal tokens are not affected and continue to work when you make 2FA required.
+However, when users [enable 2FA](/docs/platform/howto/user-2fa) their existing tokens
```

Comment:
Both cases are might.... 

However, the token might no longer work to access the organization's data or when users [enable 2FA](/docs/platform/howto/user-2fa) their existing tokens might stop working.

Reply:
`Personal tokens may stop working when 2FA is enforced.` ? To keep this as straightforward as possible.